frontend/components/ShareButton/index.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import ShareIcon from '@shopgate/pwa-ui-ios/icons/ShareIcon';
+import MaterialShareIcon from '@shopgate/pwa-ui-material/icons/MaterialShareIcon';
+import Ripple from '@shopgate/pwa-ui-shared/Ripple';
+import PropTypes from 'prop-types';
+import styles from './style';
+import getConfig from '../../helpers/getConfig';
+import connect from '../../connector';
+
+const { androidShareSVG } = getConfig();
+const { iOSShareSVG } = getConfig();
+/**
+ * ShareButton component
+ */
+class ShareButton extends Component {
+  static propTypes = {
+    iOSTheme: PropTypes.func.isRequired,
+    shareItem: PropTypes.func.isRequired,
+    shareParams: PropTypes.shape(),
+  };
+
+  static defaultProps = {
+    shareParams: null,
+  };
+  /**
+   * Handles the share button click
+   * Show's share screen for app
+   * @param {Object} event The click event object
+   */
+  handleClick = () => {
+    this.props.shareItem();
+  }
+  /**
+   * Renders the share icon depending on theme
+   * @returns {JSX}
+   */
+  renderIcon() {
+    if (this.props.iOSTheme()) {
+      return <ShareIcon svg={iOSShareSVG} />;
+    }
+    return <MaterialShareIcon svg={androidShareSVG} />;
+  }
+  /**
+   * Renders the components
+   * @returns {JSX}
+   */
+  render() {
+    if (!this.props.shareParams) {
+      return null;
+    }
+    const buttons = this.props.iOSTheme() ? styles.iOSButtons : styles.androidButtons;
+    const className = this.props.iOSTheme() ? styles.buttonFlat : styles.button;
+    return (
+      <div className={`${buttons}`}>
+        <button
+          className={`${className}`}
+          onClick={this.handleClick}
+          data-test-id="shareIcon"
+        >
+          <Ripple className={`${styles.ripple}`}>
```

Comment:
In order to make  the ripple effect visible, we might want to marry `onComplete` Ripple prop with button `onClick`.
Or just set a timeout to button `onClick` so the ripple is finished.

Check a `FavoritesButton` component in ui-shared for reference.

Reply:
Married OnComplete and onClick